extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -340,14 +343,14 @@ export class ResourceTypePickerDialog extends DialogBase {
 				return rowData;
 			}
 		});
-		this.enableUiControlsWhenNotInstalling(!tool.isInstalling); // if installing the disableContainers else enable them
+		this.toggleUiControls(tool.status !== ToolStatus.Installing); // if installing then disable ui controls else enable them
 	}
 
-	private enableUiControlsWhenNotInstalling(enabled: boolean): void {
-		this._cardGroup.enabled = enabled;
-		this._agreementContainer.enabled = enabled;
-		this._optionsContainer.enabled = enabled;
-		this._dialogObject.cancelButton.enabled = enabled;
+	private toggleUiControls(enable: boolean): void {
```

Comment:
nit: But toggle usually means flipping between values. This is more of a setUiControlsEnabled

Reply:
or disabled depending on the parameter value. How about enableDisableUiControls?

Reply:
I prefer prefixing with set since it's setting a value. The enabled part is referring to the name of the property so I see no reason to change it.